src/RA_LeaderboardPopup.cpp

```diff
@@ -232,24 +233,26 @@ _Use_decl_annotations_ void LeaderboardPopup::Render(ra::ui::drawing::ISurface&
                         m_pScoreboardSurface->LoadFont(FONT_TO_USE, FONT_SIZE_TEXT, ra::ui::FontStyles::Normal);
 
                     // background
-                    m_pScoreboardSurface->FillRectangle(0, 0, m_pScoreboardSurface->GetWidth(),
-                                                        m_pScoreboardSurface->GetHeight(), nColorBackground);
-                    m_pScoreboardSurface->FillRectangle(2, 2, m_pScoreboardSurface->GetWidth() - nShadowOffset,
-                                                        m_pScoreboardSurface->GetHeight() - nShadowOffset, nColorBlack);
-                    m_pScoreboardSurface->FillRectangle(0, 0, m_pScoreboardSurface->GetWidth() - nShadowOffset,
-                                                        m_pScoreboardSurface->GetHeight() - nShadowOffset,
+                    const auto iWidth = gsl::narrow_cast<int>(m_pScoreboardSurface->GetWidth());
+                    const auto iHeight = gsl::narrow_cast<int>(m_pScoreboardSurface->GetHeight());
```

Comment:
suggest `nWidth` and `nHeight`. I don't think we use `i` as a prefix anywhere.

Reply:
sure